### DISCUSSION TITLE
Another annoying bug - Links not clickable, restart of Vienna required to fix

### DISCUSSION THREAD

**Discussion by slammer99uk** [ViennaRSS/vienna-rss#1762] [General]
As title, All of a sudden links in articles fail to work until a restart of Vienna is made. At this point the links start working again but shortly after the same behaviour returns.

It is just like the links are no longer  clickable, even to the point where the cursor does not change to show a link is clickable. This is experienced on a number of feeds where clickable links are supposed to play video content, or redirect to other pages etc.

As far as I am aware, no changes made at my end. V3.90 again

**Comment by barijaona** [comment]
1/ In "Folder" menu, which setting do you have : "Use Current Style for Articles" or "Use Web Page for Articles" ?

2/ If you click on another article in article list, does the article view update ?

**Reply by slammer99uk** [reply]
1. Using "web page view", this is the only way to get the articles to appear in full in the 3rd (article) pane.
2. Yes the article view does update but the links/media are not clickable, when the problem appears.

Am going to post a couple of screenshots for illustration

Clickable
![1BED25E4-12F1-4C3D-A4DB-2F855BDB401C_1_102_o](https://github.com/ViennaRSS/vienna-rss/assets/163581434/9793a52e-1bd3-4999-a807-51a47bb1d11f)

Not clickable
![FB172896-0CCD-4285-AED9-1F1F3D61F523_1_102_o](https://github.com/ViennaRSS/vienna-rss/assets/163581434/8f21489d-de56-4895-95a9-4de07767c33a)

**Comment by barijaona** [comment]
Can you provide me the feed URL ?
I understand that you use vertical layout ?

**Reply by slammer99uk** [reply]
No vertical, after testing it occurs in both.

BBC news feed https://feeds.bbci.co.uk/news/rss.xml

**Comment by barijaona** [comment]
I was able to reproduce the problem by :
- choosing "web page view" for this thread
- selecting an article, going to another, going back to the first one…

After a few going back and forth, the links in the article view were not active anymore. Switching between vertical/horizon layout, changing style, going to another feed worked as expected, but did not change the fact that the article view did not react to interactions.

**Reply by barijaona** [reply]
it's not 100% reproducible, though…

**Reply by slammer99uk** [reply]
Webpage view is required for showing links and media playback controls. (BBC website)

Unfortunately it is 100% reproducible here. The problem is definitely with Vienna as I have tested with at least 3 other RSS clients, Readkit, News Explorer, Netnewswire, and the problem does not exist in any of those :(

In addition, I do not have to select an article, go to next article and go back. This can occur simply by going to the next article.

I am grateful that you have taken the time, and been able to reproduce the problem. There may be hope for a fix? :)

**Reply by Darren-Kerr** [reply]
I agree with @slammer99uk, this is 100% reproducible here also.

As the initial post states, links are not clickable until you restart the Vienna app. Links then become clickable for a short period of time, then revert to not being clickable again.

This happens with all feeds that I have, which includes the BBC News feed as the OP posted (https://feeds.bbci.co.uk/news/rss.xml). I also have the 'Use Web Page for Articles' option selected in the Folder menu.

It's not just the links that become non-clickable, either. I used to be able to highlight text in an article and copy it. All that also stops working too.

Vienna : 3.9.1
MacOS Sonoma : 14.5

**Reply by slammer99uk** [reply]
Thank god it isn't just me. I have noticed that it has gotten worse lately, to the point where it takes a number of restarts of Vienna to get clickable elements back again.

Is there any hope of a fix perhaps :)

**Reply by slammer99uk** [reply]
It seems like the problem has now been fixed, Can anyone else confirm please?

**Reply by Darren-Kerr** [reply]
Negative, I still have the issue.

What version of the app are you running?
I am running 3.9.1, which I believe is the latest release.

**Reply by slammer99uk** [reply]
Same 3.9.1 I have done nothing different my end, so at a bit of a loss to say why it now works.

**Reply by josh64x2** [reply]
Which macs are you experience these issues on? It might be related to a graphics issue that only affects certain models etc.

That would then help us try and solve the issue.

Thanks!

**Reply by Darren-Kerr** [reply]
Mac Mini M2 Pro (2023)
MacOS Sonoma 14.5
Vienna 3.9.1 - Which I installed and maintain using Homebrew. I've no idea if this would make a difference, though.

**Reply by slammer99uk** [reply]
In my case iMac 27” 2020 3.8 GHz 8 core i7 - AMD Radeon Pro 5500 XT 8 GB - 132GB

**Reply by barijaona** [reply]
I cannot be sure, as I was not 100% able to reproduce the issue, but I am confident PR #1782 might improve things.
I expect to issue Vienna 3.9.2, which should include this change, next week.

**Reply by Darren-Kerr** [reply]
3.9.2 does not fix the issue.
Still unable to click links, copy text, etc.

**Reply by slammer99uk** [reply]
Still working ok here

**Reply by Darren-Kerr** [reply]
So after initially installing the Vienna 3.9.2 update, I was still experiencing the issue. Several days later, I installed the macOS Sonoma 14.6.1 update, which required a computer restart.

I am now happy to report that I have been using Vienna 3.9.2 for almost a week now without issue.
